test/compilable/test6777.d

```diff
@@ -0,0 +1,11 @@
+struct S {}
+
+class C {
+    S s;
+    alias s this;
+}
+
+void main() {
```

Comment:
Curly braces on their own lines.

Reply:
FWIW we have never really enforced the style guide for the test cases as it isn't code that gets looked at a lot (though of course it's a fair point).